Jenkinsfile

```diff
@@ -25,29 +25,21 @@ pipeline {
             '''
           }
         }
-        stage('Polkadot') {
-          steps {
-            sh '''
-              make deps-polkadot
-            '''
-          }
-        }
```

Comment:
Don't we still want to test it? Or is it just way too expensive to run?

Reply:
I added the tests back in.